src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptions.cs

```diff
@@ -48,5 +48,20 @@ public class JaegerExporterOptions
         /// Gets or sets the tags that should be sent with telemetry.
         /// </summary>
         public IEnumerable<KeyValuePair<string, object>> ProcessTags { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not a batch should be sent to the Jaeger agent for each service. Default value: true.
```

Comment:
Given there is a concern that this causes exporter to send "incorrect" data, should we make this default to "false", so that SDK by default exports correct data. If a user want to send "incorrect" data to have better UI experience, let them explicitly opt-in to this behavior?

Reply:
I think the incorrect behavior should be removed altogether, without options to enable it. The Jaeger UI enhancement is already being added in https://github.com/jaegertracing/jaeger-ui/pull/659

Reply:
I'm going to verify everything works with the new Jaeger UI and then I'll open a PR to remove the hack. Question, is a note in CHANGELOG recommending users upgrade Jaeger UI to restore the functionality sufficient?